Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -451,7 +451,7 @@ endif (CGAL_Qt5_FOUND AND Qt5_FOUND)
 
 feature_summary(WHAT OPTIONAL_PACKAGES_NOT_FOUND
                 INCLUDE_QUIET_PACKAGES
-                DESCRIPTION "NOTICE: Missing required packages that prevent the demo from being compiled:"
+                DESCRIPTION "NOTICE : Missing required packages that prevent the demo from being compiled:"
```

Comment:
Why would you want to remove the information about missing dependencies?

It helps see if there exist a platform where all dependencies are installed.

Reply:
Because I don't think any platform has ALL of the dependencies installed at once, and so the whole line would turn blue. I don't want that.

Reply:
`if (TESTSUITE_RUNNING)`? (don't know the precise syntax)

Reply:
It doesn't change anything to have this whitespace or not if we are not inside the testsuite

Reply:
Actually, I do not see the issue with having that line fully blue. If you manage to get it fully blue, then that would mean that warnings and errors are gone (because otherwise the yellow and red colors have the priority). And one it is fully blue, we can try add dependencies to a few test platforms, to have green cells. I really see any benefit in this patch. TO me, that looks like removing tests because there are red cells.

But maybe I am wrong, and you should explain why that matters.

Reply:
Well, at a timùe we are trying to get as green cells as possible in the testsuite, I am not encline to add some blue. And don't expect to see any green cell in that line if I leave it like that, as dependencies like OpenGR come with a bagful of warnings. To me, that will give the impression that most of the platforms don't test the demo, and the very few that do show it doesn't work correctly, when it is not the point.

Reply:
What is true is that originally, the blue cells of 3D demos was when Qt5 was missing. Now that our big 3D demos have a lot of features, and tones of dependencies, we have a lot more opportunities for missing optional dependencies.